docs/source/azure/policy/resources/networkinterface.rst

```diff
@@ -42,4 +42,16 @@ and then perform the delete operation on those ready for deletion.
           - type: marked-for-op
             op: delete
         actions:
-          - type: delete
\ No newline at end of file
+          - type: delete
```

Comment:
Can you please add it to the available filters list?

Reply:
I think information about errors for non-attached NICs or turned off VMs is also valuable.